src/Azure/Azure.Quantum.Client/Microsoft.Azure.Quantum.Client.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
(feel free to let me know if you'd prefer me to revert this change)
Looks like this is a remainder of the rebasing/merging epopee. This changes the first few bytes (not visible in the diff) that determine the file encoding.

Reply:
In general, the first two bytes should not be 0xFEFF, as that is the UTF-16 and UTF-32 byte order mark; its presence in a UTF-8 encoded file is a nonstandard Windows extension and reduces portability.

Reply:
I see that this particular file is ASCII-encoded (the first 2 chars are 3C 50, which are the first 2 visible characters `<` and `P`).
I'll double-check the other similar changes.

Reply:
All such changes have no invisible chars in the beginning.